addons/website/views/website_templates.xml

```diff
@@ -243,6 +243,8 @@
             </t>
         </t>
         <link t-if="request and website" rel="canonical" t-att-href="website._get_canonical_url(canonical_params=canonical_params)"/>
+
+        <link rel="preconnect" href="https://fonts.gstatic.com/" crossorigin=""/>
```

Comment:
Why not on fonts.odoocdn.com? Do we load google fonts by default?

Reply:
Yes, we load fonts from google directly for all website fonts (and for the v13.0, users can select any font from google fonts).

We could probably add a preconnect to odoocdn too for the Noto support font... not sure.